index.bs

```diff
@@ -41,6 +41,7 @@ Interfaces:
 * {{AbortSignal}} [[!DOM]]
 * {{Blob}} [[!FILEAPI]]
 * {{ByteLengthQueuingStrategy}} [[!STREAMS]]
+* {{CloseEvent}} [[!HTML]]
```

Comment:
There's no `CloseEvent` interface defined in the HTML spec. `MessagePort`'s `close()` method is defined as firing a `close` event on the receiver, but that's a regular `Event`, not the `CloseEvent` interface from the WebSocket spec.